packages/classic/package.json

```diff
@@ -45,6 +49,9 @@
     "prepublishOnly": "./build/embed-dependencies '@progress/kendo-theme-default'",
     "postpublish": "rm -rf modules && git checkout scss"
   },
+  "peerDependencies": {
+    "@progress/kendo-theme-default": "^4.41.0"
+  },
   "devDependencies": {
     "@progress/kendo-theme-default": "^4.41.0"
```

Comment:
if the Default theme is in the peer dependencies list, do we need it in dev dependencies as well?

Reply:
yes, we need it :)